templates/security/cve/cve.html

```diff
@@ -188,6 +188,8 @@ <h2>Status</h2>
                       {% endif %}
                     </td>
                   </tr>
+                {% else %}
```

Comment:
I think the better fix would be to move the opening `tr` on line 139 above `{% if release.codename in statuses %}` and the closing `tr` on line 190 beneath the closing `{% endif %}`

Reply:
That will avoid the `else`. Fine.

Also I will remove a number of extra, redundant `{% if release.codename in statuses %}` inside the main `{% if release.codename in statuses %}`.